views/login/sin.pug

```diff
@@ -0,0 +1,28 @@
+extends ../base
+
+block content
+
+  h1 #{title}
+
+  p Now that we have your personal code, we need some additional information in order to verify that it’s you.
+
+  p Please enter your Social Insurance Number (SIN) in the field below.  
+
+    form.pure-form.pure-form-stacked(method='post')
+      .pure-u-4-5.pure-u-sm-1-2.pure-u-md-1-3
+        div
+          label(for='sin') Social Insurance Number
+          span.pure-form-message For Example: 123 456 789 
```

Comment:
Good idea. We should do some backend filtering to make sure we can handle values that look like this.